cider/util/sqldefs.h

```diff
@@ -26,7 +26,8 @@
 // in sync.
 enum SQLOps {
   kEQ = 0,
-  // BitWise Equal <=> a = b or (a is null and b is null)
+  // The only difference between EQ and BW_EQ is that BW_EQ will check nullability of both
```

Comment:
`check nullability of both sides` means `(a is null and b is null)` ?

Reply:
Seems previous comments is more clear. They should both check nullability but for `kEQ`, it doesn't return true if both sides are null?

Reply:
See [check-equal-bitwise](https://iq.opengenus.org/check-equal-bitwise/) for more information.